docs/controllers/extendedsecret.md

```diff
@@ -58,8 +82,28 @@ spec:
   - key encipherment
 ```
 
+### **_CertificateSigningRequest Controller_**
+
+![certsr-controller-flow](quarks_certsrcontroller_flow.png)
+
+#### Watches in CSR controller
+
+- `Certificate Signing Request`: Creation
+
+#### Reconciliation in CSR controller
+
+- once the request is approved by Kubernetes API, will generate a certificate stored in a Kubernetes secret, that is recognized by the cluster.
+
+#### Highlights in CSR controller
+
 The CertificateSigningRequest controller watches for `CertificateSigningRequest` and approves ExtendedSecret-owned CSRs and persists the generated certificate.
 
+## Relationship with the BDPL component
+
+![bdpl-ejob-relationship](quarks_gvc_and_esec_flow.png)
+
+The above image illustrates the interaction of the **Generated Variables** Controller with the **ExtendedSecret** Controllers. The Generated Variables Controller when reconciling, will list all variables of a BOSH manifest(basically all BOSH variables), and generate for each an ExtendedSecret instance, which will trigger the following controller, the ExtendedSecret one.
```

Comment:
Figure X illustrates the interaction of the **Generated Variables** Controller with the **ExtendedSecret** Controller. When reconciling the Generated Variables Controller lists all variables of a BOSH manifest (basically all BOSH variables), and generates an ExtendedSecret instance per variable, which will trigger the ExtendedSecret controller.